README.md

```diff
@@ -31,10 +31,11 @@ import Ember from 'ember';
 import hbs from 'htmlbars-inline-precompile';
 import connect from 'ember-redux/components/connect';
 import ajax from 'example/utilities/ajax';
+import getUsersByAccountId from '../reducers';
```

Comment:
This is a great simplistic use case! I'm wrestling with the idea of our most basic example showing this off vs adding this to the guides/gh-pages documentation (and in doing so, actually adding a new page for the connect api like you see in react-redux) -thoughts on the pros/cons of either decision ?